controllers/controllers.js

```diff
@@ -60,11 +61,26 @@ const getComments = (req, res, next) => {
     .catch(next);
 };
 
+const addComment = (req, res, next) => {
+  const id = req.params.review_id;
+  const { username, body } = req.body;
+  console.log(username, body);
+
+  fetchReviewId(id)
+    .then(() => {
+      postComment(id, username, body)
+        .then((comment) => res.status(201).send(comment))
+        .catch(next);
```

Comment:
you have a nested `.then()` here, could this be refactored so they are chained instead please